repl2.html

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+permalink: /repl2/
+fullscreen: true
+third_party_js:
+- https://unpkg.com/babel-standalone@^6
+- https://unpkg.com/react@16.0.0-beta.3/umd/react.production.min.js
+- https://unpkg.com/react-dom@16.0.0-beta.3/umd/react-dom.production.min.js
+- https://unpkg.com/codemirror@5.28.0/lib/codemirror.js
+- https://unpkg.com/codemirror@5.28.0/mode/javascript/javascript.js
+- https://unpkg.com/codemirror@5.28.0/mode/xml/xml.js
+- https://unpkg.com/codemirror@5.28.0/mode/jsx/jsx.js
+- https://unpkg.com/codemirror@5.28.0/keymap/sublime.js
+- https://unpkg.com/codemirror@5.28.0/addon/display/placeholder.js
+- https://unpkg.com/codemirror@5.28.0/addon/selection/active-line.js
+- https://unpkg.com/lz-string@1.4.4/libs/base64-string.js
+- https://unpkg.com/lz-string@1.4.4/libs/lz-string.min.js
```

Comment:
I forgot to mention - I changed all of these to exact versions in order to get more efficient caching. With a version range, unpkg redirects to the latest version matching the range, and the redirect is only cached for 5 minutes to avoid caching an old version for a long period of time. That means that basically every time you load the REPL, it'd need to send a network request and do the redirect. With an exact version, it can be cached "forever" and loaded directly from cache, totally avoiding the network request if the files have already been cached.

Exact versions also ensures that the live site won't suddenly break (in the same way that a Yarn lockfile ensures a local build won't break)

Only exception is for babel-standalone because we do always want the very latest version.